gradle/libs.versions.toml

```diff
@@ -14,7 +14,7 @@ ktor = "2.3.11"
 
 ## Analysis
 kotlin-compiler = "2.0.20"
-kotlin-compiler-k2 = "2.1.0-dev-5441"
+kotlin-compiler-k2 = "2.1.0-dev-7103"
```

Comment:
@vmishenev have you checked, by the way, if the behavior on latest available version of AA is the same?

Reply:
The same(